src/modules/beamos/config

```diff
@@ -6,17 +6,20 @@
 TRUE_USER=pi
 
 # Branches
-[ -n "$BEAMOS_MRBEAMPLUGIN_REPO_BRANCH" ] || BEAMOS_MRBEAMPLUGIN_REPO_BRANCH=mrbeam2-stable
+## Plugins
+[ -n "$BEAMOS_MRBEAMPLUGIN_REPO_BRANCH" ] || BEAMOS_MRBEAMPLUGIN_REPO_BRANCH=v0.11.0
```

Comment:
This section changes with each new release for BeamOS, right ?

Reply:
yes, I create a ticket to automate this with a github release

Reply:
issue created: https://mr-beam.atlassian.net/browse/SW-1573